src/server/game/Conditions/ConditionMgr.cpp

```diff
@@ -1559,8 +1559,8 @@ bool ConditionMgr::isSourceTypeValid(Condition* cond)
                 return false;
             }
             break;
-        case CONDITION_SOURCE_TYPE_UNUSED_XX:
-            sLog->outErrorDb("CONDITION_SOURCE_TYPE_UNUSED_XX (20) is not in use.");
+        case CONDITION_SOURCE_TYPE_UNUSED_20:
+            sLog->outErrorDb("CONDITION_SOURCE_TYPE_UNUSED_20 is not in use.");
```

Comment:
What do you think about this, should I word it different from "is not in use." or is it ok?

Reply:
@Kitzunu almost :)

```
sLog->outErrorDb("CONDITION_SOURCE_TYPE_UNUSED_20 is not in use (%u), skipped", cond->SourceEntry);
```

so we show which `SourceEntry` is causing the problem

Reply:
Could you teach me where (%u) comes from?

Reply:
Also isn't it better to have "CONDITION_SOURCE_TYPE_UNUSED_20 is not in use SourceEntry = (%u), skipped" or does that come in %u?

Reply:
`%u` in this case it's just giving you the value of `cond->SourceEntry`

![image](https://user-images.githubusercontent.com/75517/79028231-7e6f8900-7b8f-11ea-9e50-ef6107c1b16f.png)

more about it: https://en.wikipedia.org/wiki/Printf_format_string

feel free to ask me if you need more info :)

Reply:
Ok I think I understand, will update it soon. 
Do you agree with the version I said above?

Reply:
@Kitzunu yes! ;)